### PR TITLE
Initial version of website/docs portal

### PR DESCRIPTION
**Pull request by wtrocki**
The motivation for this PR is to continue working on website design and show how we can 

Example of the functional websites: https://v2.docusaurus.io/showcase

For the moment this is basic docusaurus template - I'm blocked by moving the docs files to some folder first.

### CODE REVIEW

File: website/blog/2019-05-28-hola.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (mhdawson) [inline website/blog/2019-05-28-hola.md:1]: Should these files be here?

Comment (mhdawson) [reply website/blog/2019-05-28-hola.md:1]: @lholmquist I probably should have asked earlier but does using docusaurus for this fit into our Nodedeshift website ?

Comment (wtrocki) [reply website/blog/2019-05-28-hola.md:1]: Those are for testing. I see now that nodeshift website can be used for hosting thouse so we might use what is used there.

**Comment by wtrocki** [comment]
I did not know that this goes to nodeshift website. Docusaurus is an node.js and react based documentation system. Nodeshift uses https://jekyllrb.com so there is no compatibility here.

**Comment by wtrocki** [comment]
Closing as this is not going to work with current website. happy to discuss it on next call

**Pull request closed without merging**